backend/core/src/primary-adapters/rest/video/helpers/string-to-enum-helper.ts

```diff
@@ -0,0 +1,5 @@
+import { VideoPrivacy } from 'shared/build';
+
+export const stringToEnumHelper = (str: string): VideoPrivacy => {
+  return VideoPrivacy[str as unknown as VideoPrivacy];
```

Comment:
rename, please, to more fitting name, according to its purpose: like stringToVideoPrivacy(Enum)(Helper), along with the filename